drivers/gpio/Kconfig.mcux_rgpio

```diff
@@ -0,0 +1,12 @@
+# MCUX RGPIO configuration options
+
+# Copyright (c) 2023, NXP
```

Comment:
We are generally trying to format copyright statements as `Copyright 2023, NXP`. Is this format OK with you?

Reply:
replaced with new statement.